api/v1/routes/plans.py

```diff
@@ -0,0 +1,22 @@
+from fastapi import FastAPI, Depends, HTTPException, status, APIRouter
+from sqlalchemy.orm import Session
+from api.v1.models.plans import SubscriptionPlan
+from api.v1.schemas.plans import CreateSubscriptionPlan, SubscriptionPlanResponse
+from api.db.database import get_db
+from api.utils.dependencies import get_current_admin
+
+plans = APIRouter(prefix="/api/v1", tags=["Plans"])
+
+@plans.post("/plans", response_model=SubscriptionPlanResponse, status_code=status.HTTP_201_CREATED)
+def create_subscription_plan(plan: CreateSubscriptionPlan, db: Session = Depends(get_db), current_admin: str = Depends(get_current_admin)):
+    # Check to see if the plan already exist
```

Comment:
Read the get current user tutorial I dropped in the channel, also the `current_admin` should be a `User` object, why are you using type `str`

Reply:
I have resolved this following the Fast API documentation